modules/labs-react/combobox/stories/stories.tsx

```diff
@@ -1,7 +1,11 @@
 import React, {useState, ReactNode, ReactElement, FC, ChangeEvent} from 'react';
 import {storiesOf} from '@storybook/react';
 
-import {Combobox, ComboboxProps, ComboBoxMenuItemGroup} from '@workday/canvas-kit-labs-react';
```

Comment:
how did this not blow up before... lol

Reply:
The difference is that imports in stories don't matter. These don't have to change, but I made the lint rule catch them anyway.